creation/templates/gwms-renew-proxies.timer

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Hourly frontend and pilot proxy renewal
+After=syslog.target network.target
+Wants=network.target
+
+[Timer]
+OnActiveSec=0s
+OnUnitActiveSec=1h
+Persistent=true
```

Comment:
I typically request that `RandomizedDelaySec` to be set for timer units (I'd suggest 5 minutes is appropriate here).  That helps prevent things from synchronizing across multiple clients.

Here, I expect there are so few clients it seems pretty optional.  Couldn't hurt?

Reply:
Also, from the docs for `Persistent`:

> Note that this setting only has an effect on timers configured with OnCalendar=.

So please remove this line.